src/web/core/src/main/java/org/geoserver/web/services/BaseServiceAdminPage.java

```diff
@@ -70,6 +73,7 @@
  */
 // TODO WICKET8 - Verify this page (and derived pages?) work OK
 public abstract class BaseServiceAdminPage<T extends ServiceInfo> extends GeoServerSecuredPage {
+    public static final String WORKSPACE_ADMIN_SERVICE_ACCESS = "WORKSPACE_ADMIN_SERVICE_ACCESS";
```

Comment:
A bit of javadoc would not hurt

```suggestion
    /** Allows workspace admins access to the workspace service configuration too **/
    public static final String WORKSPACE_ADMIN_SERVICE_ACCESS = "WORKSPACE_ADMIN_SERVICE_ACCESS";
```

Reply:
done